src/expr/subtype_elim_node_converter.cpp

```diff
@@ -23,7 +25,7 @@ SubtypeElimNodeConverter::SubtypeElimNodeConverter() {}
 
 bool SubtypeElimNodeConverter::isRealTypeStrict(TypeNode tn)
 {
-  return tn.isReal() && !tn.isInteger();
+  return tn.isReal();
```

Comment:
this is because integer is no longer a subtype of real?

Reply:
Yes